osu.Game/Screens/OnlinePlay/Lounge/LoungeSubScreen.cs

```diff
@@ -237,10 +241,16 @@ protected virtual IEnumerable<Drawable> CreateFilterControls()
 
             statusDropdown.Current.BindValueChanged(_ => UpdateFilter());
 
-            yield return statusDropdown;
-        }
+            publicityDropdown = new SlimEnumDropdown<RoomPublicityFilter>
```

Comment:
Should this filter be added as an override to `MultiplayerLoungeSubScreen` instead of being included here in the base class? It is currently impossible to create a password protected playlist, making this filter redundant.

Reply:
I would say so, a dropdown shouldn't exist in a place where it can't function.